ingestion/tests/magen_ingestion_test.py

```diff
@@ -2218,11 +2233,13 @@ def test_post_file_share_Fail_No_PublicKey_Multiple_receivers(self):
                                              json_body=ks_get_resp_json_obj,
                                              response_object=None)
             get_mock = Mock(return_value=get_rest_return_obj)
+            mock_value = Mock(return_value=MongoReturn(count=1))
             with patch('magen_rest_apis.rest_client_apis.RestClientApis.http_get_and_check_success', new=get_mock):
-                jquery_file_share_url = server_urls_instance.ingestion_server_base_url + "file_share/"
-                file_share_resp_obj = type(self).app.post(jquery_file_share_url, data={'file': share_asset_id,
-                                                                                       'selected_user': receivers},
-                                                          headers={'content-type': 'multipart/form-data'})
+                with patch('magen_user_api.user_model.UserModel.select_by_email', new=mock_value):
```

Comment:
You have repeated this chunk about 5 times now. Let's wrap it in a function where you can provide parameters and reduce code repeting